.remarkrc.js

```diff
@@ -0,0 +1,12 @@
+// File for setting up the markdown linter configuration.
+// Add the required plugin to the plugins array.
+
+const preset = {
+  plugins: [
+    'remark-lint-no-dead-urls',
+    'remark-validate-links',
+    'remark-lint-code-block-style'
```

Comment:
Does it just check between <code>```</code> and indent code block format?

We do not need it since most users don’t know about indent based code blocks.

I told you about a different thing. If you have a code block in *the list* you need to add indent to <code>```</code> block.

Wrong:

<pre>
1. List item
```
code
```
3. List item
</pre>

Right:

<pre>
1. List item
   ```
   code
   ```
3. List item
</pre>